Modules/_zoneinfo.c

```diff
@@ -1200,14 +1199,14 @@ calendarrule_year_to_timestamp(TransitionRuleType *base_self, int year)
     }
 
     int64_t ordinal = ymd_to_ord(year, self->month, month_day) - EPOCHORDINAL;
-    return ((ordinal * 86400) + (int64_t)(self->hour * 3600) +
+    return ((ordinal * 86400L) + (int64_t)(self->hour * 3600L) +
             (int64_t)(self->minute * 60) + (int64_t)(self->second));
```

Comment:
I prefer syntax (int64_t)var * constant, rather than trying to declare the constant as an int64_t or a long.

For example, `(int64_t)self->hour * 3600` rather than `(int64_t)(self->hour * 3600L)`.